File: src/sample.py
# Sample Python file with intentional lint errors
import os
import json

unused_var = 42
another_unused = "test"


def greet(name):
    x = 10
    if name == "world":
        print("Hello, " + name)
    return name




result = greet("world")
print(result)
dead_code = 999
another_dead = "hook test v4"

File: src/sample.ts
// Sample TypeScript file with intentional lint errors

const unusedVariable = "this is never used";

function greet(name: string) {
  
  var x = 42;
  if (name == "world") {
    console.log("Hello, " + name);
  }
  return x;
}

const result = greet("world");
console.log(result);
// trigger hook test v6 - full linter hook
var unused = 123;

console.log(void 0 === undefined);
